Source/Core/Core/PowerPC/JitCommon/JitBase.cpp

```diff
@@ -58,6 +58,13 @@ void JitBase::RefreshConfig()
   m_fastmem_enabled = Config::Get(Config::MAIN_FASTMEM);
   m_mmu_enabled = Core::System::GetInstance().IsMMUMode();
   m_pause_on_panic_enabled = Core::System::GetInstance().IsPauseOnPanicMode();
+  m_accurate_cpu_cache_enabled = Config::Get(Config::MAIN_ACCURATE_CPU_CACHE);
+  if (m_accurate_cpu_cache_enabled)
+  {
+    m_fastmem_enabled = false;
```

Comment:
Does anything extra need to be done to accommodate SoftMMU in JitArm64? @JosJuice

Reply:
Oh yeah, now that you mention it, I don't think this will work with soft MMU as it's currently implemented. (Maybe we could implement the cache handling in soft MMU in the future, but that would be a lot of work...)

@TheLordScruffy, in addition to what you're currently doing, in `JitArm64::EmitBackpatchRoutine`, please force `emit_fastmem` to `false` and `emit_slowmem` to `true` when your new setting is enabled.